### ISSUE TITLE
Can't trigger release if there's only one commit

### ISSUE THREAD

**Issue by egoist** [algolia/shipjs#945]
**Describe the bug**
with only one commit, e.g. `feat: init`, it says:

```
› Getting a revision range for this release.
Git tag 'v0.0.0' doesn't exist.
? From which commit do you want to release? 1) 31b86aa (HEAD -> main, origin/main) feat: init
  31b86aa..HEAD
› Calculating the next version.
Nothing to release!
```

**To Reproduce**
1. create a repo
2. create a commit
3. run `shipjs prepare`

**Expected behavior**

**Screenshots**

**Environment (please complete the following information):**
 - environment: (your local machine, CircleCI, Travis CI, etc.)
 - `node -v`: 
 - `npm -v` or `yarn -v`: 
- Ship.js version: 0.23.0

**Additional context**

**Comment by eunjae-lee** [comment]
Yeah it's related, but if there is only one commit, that will happen because there is no commit to release *since* the commit you chose. Unfortunately the behavior we have is like that. We'll try to find a way to fix this, but in the meantime you can still release it if there is one more commit.

**Comment by egoist** [comment]
@eunjae-lee  OK, thanks for the info.
